shardingsphere-integration-test/shardingsphere-test-suite/src/test/resources/integrate/cases/dcl/dcl-integrate-test-cases.xml

```diff
@@ -41,7 +41,6 @@
     <dcl-test-case sql="ALTER USER user_dev WITH PASSWORD 'passwd_dev'" db-types="PostgreSQL"/>
     <dcl-test-case sql="ALTER USER user_dev WITH PASSWORD 'passwd_dev'" db-types="PostgreSQL"/>
     <dcl-test-case sql="ALTER ROLE role_dev_bak DROP MEMBER member1" db-types="SQLServer"/>
-    <dcl-test-case sql="ALTER USER CURRENT_USER WITH ENCRYPTED PASSWORD 'password'" db-types="PostgreSQL"/>
```

Comment:
Hi thanks for your clarification at #6943.
Got it. I think MySQL TESTS have the same issue as well. My suggestions are,
First, could you look at how MySQL tests deal with this issue in the same test-case file?
Second, do you think it is better to test the relevant SQL case with another new user or role?